packages/appChrome/stories/AppChrome.stories.tsx

```diff
@@ -42,6 +43,21 @@ storiesOf("AppChrome", module)
       <SidebarBareContent />
     </Sidebar>
   ))
+  .addWithInfo("Sidebar bare w/ custom background", () => (
+    <Sidebar backgroundColor={greenDarken1} isOpen={true}>
+      <SidebarBareContent />
+    </Sidebar>
+  ))
+  .addWithInfo("Sidebar bare w/ custom padding", () => (
+    <Sidebar paddingLeft="xl" paddingRight="xl" isOpen={true}>
```

Comment:
I would really really love to see the usage of [storybook-knobs](https://github.com/storybooks/storybook/tree/master/addons/knobs#array) here. It would allow us to play with it and get some more insights on how it behaves. It would also allow us to do most of these examples in one bigger, more configurable one, which would be awesome.

If you like we can pair on this 👍

Reply:
I would say yes, as you are adding a lot of examples which may be cluttered and hard to oversee (hurting usability), but you judge :)

Reply:
+1 for knobs usage. Also happy to pair with you on that